disperser/apiserver/disperse_blob_v2.go

```diff
@@ -32,6 +33,11 @@ func (s *DispersalServerV2) DisperseBlob(ctx context.Context, req *pb.DisperseBl
 		return nil, err
 	}
 
+	// Check against payment meter to make sure there is quota remaining
+	if err := s.checkPaymentMeter(ctx, req); err != nil {
```

Comment:
Separate out rate limiting checking and place it **after** the param validation, because this will make external RPC calls